### PR TITLE
Adding pt-br translation to 0.85/0.86 version

### PR DESCRIPTION
**Pull request by Plugner**
#### Description: 

Remove English messages from pt-BR translation.
____

By making this pull request, I represent that I have the right to waive copyright and related rights to my contribution, and agree that all copyright and related rights in my contributions are waived, and I acknowledge that the TownyAdvanced organization has the copyright to use and modify my contribution under the Towny [License](https://github.com/LlmDl/Towny/blob/master/LICENSE.md) for perpetuity.

### CODE REVIEW

File: resources/pt-br.yml [hunk 1]
```diff
@@ -29,9 +29,9 @@ description: >
 # Amarelo = &e           Branco = &f
 
 
-default_towny_prefix: '&6[Towny] &b'
-default_town_prefix: '&6[%s] &b'
-default_nation_prefix: '&6[%s] &6'
+default_towny_prefix: '&9Towny> &7'
+default_town_prefix: '&6Cidade> &b[%s] &7'
+default_nation_prefix: '&2Nação> &b[%s] &7'
```
Comment (LlmDl) [inline resources/pt-br.yml:34]: These lines should be brought back to the normal format. This looks like something your own server does to customize Towny but I doubt every Brazilian server will want it this way.

**Review by LlmDl — CHANGES_REQUESTED** [review]

File: resources/pt-br.yml (hunk 1 above)
Comment (Plugner) [reply resources/pt-br.yml:34]: Oops, sorry, this fork is to my server, and I forgot to adjust this.

**Comment by Plugner** [comment]
Done

File: resources/pt-br.yml [hunk 2]
```diff
@@ -29,7 +29,7 @@ description: >
 # Amarelo = &e           Branco = &f
 
 
-default_towny_prefix: '&6[Towny] &b'
+default_towny_prefix: '&6[Towny] &b &7'
```
Comment (LlmDl) [inline resources/pt-br.yml:32]: Still a bit of custom formatting to undo.

**Review by LlmDl — APPROVED** [review]

**Comment by LlmDl** [comment]
There's now a version 0.87 which isn't final yet but will require some more translation.

**Pull request merged**